words/words.md

```diff
@@ -47,27 +47,27 @@ Also schnall dich an: Wir erwarten einige Turbulenzen.
     </div>
 </div>
 
-Piloten und Pilotinnen benutzen Flugsimulatoren, um zu lernen, wie man fliegt.
+Piloten und Pilotinnen benutzen Flugsimulatoren, um zu lernen, wie man ein Flugzeug wieder heil landet.
```

Comment:
würde inhaltlich näher am Original bleiben aber die Struktur etwas vereinfachen: 
"Pilots use flight simulators to learn how not to crash planes." 
-> "Piloten und Pilotinnen lernen in Flugsimulatoren wie man ein Flugzeug vor dem Absturz bewahrt."

Reply:
Hab auch auch nichts dagegen da noch ein "um zu lernen wie" reinzuschieben, aber ich versuche wo möglich die zusätzliche Länge im Deutschen auszugleichen.

Reply:
Vielleicht eine etwas einfachere Satzstruktur: 
Pilotinnen und Piloten trainieren mit Flugsimulatoren, um künftige Abstürze zu verhindern.

Reply:
@herrhundt finde ich super!